%%%%%%%%%%%%%%%%%%
%% PUBLICATIONS %%
%%%%%%%%%%%%%%%%%%

\section{Complete List of Publications}

% \subsection{Peer Reviewed Publications}

%%%%%%%%%%
%% 2022 %%
%%%%%%%%%%

\subsection{2022}

    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
    
    	Mashanov V., Whaley L., Davis K., Heinzeller T., Machado D.J., Reid R.W., Kofsky J., and Janies D.A. (2022) A subterminal growth zone at arm tip likely underlies life-long indeterminate growth in brittle stars. \textit{Frontiers in Zoology}, \textbf{19}: 15. DOI: \href{https://doi.org/10.1186/s12983-022-00461-0}{10.1186/s12983-022-00461-0}.
    	
    	Ford C.T., Machado D.J., and Janies D.A. (2022) Predictions of the SARS-CoV-2 Omicron variant (B.1.1.529) spike protein receptor-binding domain structure and neutralizing antibody interactions. \textit{Frontiers in Virology}, \textbf{2}: 830202. DOI: \href{https://doi.org/10.3389/fviro.2022.830202}{10.3389/fviro.2022.830202}.
    
    \end{hangparas}}

%%%%%%%%%%
%% 2021 %%
%%%%%%%%%%

\subsection{2021}
    
    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}

		Machado D.J., White III R.A., Kofsky J., and Janies D.A. (2021) Fundamentals of genomic epidemiology, lessons learned from the coronavirus disease 2019 (COVID-19) pandemic, and new directions. \textit{Antimicrobial Stewardship \& Healthcare Epidemiology}, \textbf{1}: E60. DOI: \href{https://doi.org/10.1017/ash.2021.222}{10.1017/ash.2021.222}.
		
		Machado D.J., Scott R., Guirales S., and Janies D.A. (2021) Fundamental evolution of all \emph{Orthocoronavirinae} including three deadly lineages descendent from Chiroptera‐hosted coronaviruses: SARS‐CoV, MERS‐CoV and SARS‐CoV‐2. \emph{Cladistics}, \textbf{37(5)}: 461-488. DOI: \href{https://doi.org/10.1111/cla.12454}{10.1111/cla.12454}.

		Machado D.J., Marques F.P.L., Jiménez-Ferbans L., and Grant T. (2021) An empirical test of the relationship between the bootstrap and likelihood ratio support in maximum likelihood phylogenetic analysis. \textit{Cladistics} (early view). DOI: \href{https://doi.org/10.1111/cla.12496}{10.1111/cla.12496}.
		
		Machado D.J., Castroviejo-Fisher S., and Grant T. (2021) Evidence of absence treated as absence of evidence: the effects of variation in the number and distribution of gaps treated as missing data on the results of standard maximum likelihood analysis. \emph{Molecular Phylogenetics and Evolution}, \textbf{154}: 106966. DOI: \href{https://doi.org/10.1016/j.ympev.2020.106966}{10.1016/j.ympev.2020.106966}.

        Orrico V.G.D., Grant T., Faivovich J., Rivera‐Correa M., Rada, M.A., Lyra M.L., Cassini C.S., Valdujo P.H., Schargel W.E., Machado D.J., Wheeler W.C., Barrio‐Amorós C., Loebmann D., Moravec J., Zina J., Solé M., Sturaro M.J., Peloso P.L.V., Suarez P., and Haddad C.F.B. (2021). The phylogeny of Dendropsophini (Anura: Hylidae: Hylinae). \emph{Cladistics}, \textbf{37(1)}: 73–105. DOI: \href{https://doi.org/10.1111/cla.12429}{10.1111/cla.12429}.
        
        Trevisan B., Machado D.J., Lahr D.J.G., Marques F.P.L.M. (2021) Comparative characterization of mitogenomes from five orders of cestodes (Eucestoda: Tapeworms). \textit{Frontiers in Genetics}, \textbf{12}: 788871. DOI: \href{ https://doi.org/10.3389/fgene.2021.788871}{10.3389/fgene.2021.788871}.

	\end{hangparas}}

%%%%%%%%%%
%% 2020 %%
%%%%%%%%%%
		
\subsection{2020}

    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
    
        Machado D.J., Schneider A.D, and Janies D. (2020) Enhanced genome annotation strategy provides novel insights on the phylogeny of \emph{Flaviviridae}. \emph{Viruses},\textbf{12(8)}: 892. DOI: \href{https://www.mdpi.com/1999-4915/12/8/892}{10.3390/v12080892}.

	    Mashanov V., Akiona J., Khoury M., Ferrier J., Reid R., Machado D.J., … Janies D. (2020) Active Notch signaling is required for arm regeneration in a brittle star. \emph{PLoS ONE}, \textbf{15(5)}: e0232981. DOI: \href{https://doi.org/10.1371/journal.pone.0232981}{10.1371/journal.pone.0232981}

	    Lin J.-P., Tsai M., Kroh A., Trautman A., Machado D.J., Chang L., … Janies D. (2020) The first complete mitochondrial genome of the sand dollar \emph{Sinaechinocyamus mai} (Echinoidea: Clypeasteroida). \emph{Genomics}, \textbf{112(2)}: 1686-1693. DOI: \href{https://doi.org/10.1016/j.ygeno.2019.10.007}{10.1016/j.ygeno.2019.10.007}.

	\end{hangparas}}
	
%%%%%%%%%%
%% 2019 %%
%%%%%%%%%%

\subsection{2019}

    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
		Trevisan B., Alcantara D.M.C., Machado D.J., Marques F.P.L. \& Lahr D.J.G. (2019) Genome Skimming is a low-cost and robust strategy to assemble complete mitochondrial genomes from ethanol preserved specimens in biodiversity studies. \emph{PeerJ}, \textbf{7}: e7543. DOI: \href{https://doi.org/10.7717/peerj.7543}{10.7717/peerj.7543}.
	\end{hangparas}}

%%%%%%%%%%
%% 2018 %%
%%%%%%%%%%

\subsection{2018}

    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
		Machado D.J., Janies D., Brouwer C. \& Grant T. (2018) A new strategy to infer circularity applied to four new complete frog mitogenomes. \emph{Ecology and Evolution}, \textbf{8(8)}: 4011-4018. DOI: \href{http://doi.wiley.com/10.1002/ece3.3918}{10.1002/ece3.3918}.
	\end{hangparas}}

%%%%%%%%%%
%% 2017 %%
%%%%%%%%%%

\subsection{2017}

    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
		Grant T., Rada M.A., Anganoy-Criollo M., Batista A., Dias P.H., Jeckel A.M., Machado D.J. \& Rueda-Almonacid J.V. (2017) Phylogenetic Systematics of Dart-Poison Frogs and Their Relatives Revisited (Anura: Dendrobatoidea). \emph{South American Journal of Herpetology} \textbf{12(s1)}: S1-S90. DOI: \href{http://www.bioone.org/doi/10.2994/SAJH-D-17-00017.1}{10.2994/SAJH-D-17-00017.1}.
	\end{hangparas}}

%%%%%%%%%%
%% 2016 %%
%%%%%%%%%%

\subsection{2016}

    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
		Machado D.J., Lyra M.L. \& Grant, T. (2016) Mitogenome assembly from genomic multiplex libraries: Comparison of strategies and novel mitogenomes for five species of frogs. \emph{Molecular Ecology Resources}, \textbf{16(3)}: DOI: 686--693.~\href{https://doi.org/10.1111/1755-0998.12492}{10.1111/1755-0998.12492}.
	\end{hangparas}}

%%%%%%%%%%
%% 2015 %%
%%%%%%%%%%

\subsection{2015}

    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
		Machado D.J. (2015) YBYRÁ facilitates comparison of large phylogenetic trees. \emph{BMC Bioinformatics}, \textbf{16(1)}: 204. DOI: \href{https://doi.org/10.1186/s12859-015-0642-9}{10.1186/s12859-015-0642-9}.
	\end{hangparas}}

%%%%%%%%%%
%% 2012 %%
%%%%%%%%%%

\subsection{2012}

    {\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
		Machado D.J. \& Marques F.P.L. (2012) The forgotten origin of~\emph{Acanthobothrium}~Blanchard, 1848 (Tetraphyllidea: Onchobothriidae). \emph{Zootaxa} \textbf{3505}: 86--88. ISSN: \href{http://www.mapress.com/zootaxa/2012/f/z03505p088f.pdf}{11755326}.
	\end{hangparas}}

%%%%%%%%%%%%%%
%% In press %%
%%%%%%%%%%%%%%
        
% \subsection{In Press}
% 	{\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}\begin{hangparas}{.25in}{1}
%
% 	\end{hangparas}}

%%%%%%%%%%%%%%%
%% Preprints %%
%%%%%%%%%%%%%%%

\subsection{Preprints}

	{
		\setlength{\parskip}{.5em}\renewcommand{\baselinestretch}{2.0}
		\begin{hangparas}{.25in}{1}
		
		Ford C.T., Scott R.,  Machado D.J, and  Janies D.A. (2021) Sequencing data of North American SARS-CoV-2 isolates shows widespread complex variants. \textit{medRxiv}. DOI: \href{https://doi.org/10.1101/2021.01.27.21250648}{2021.01.27.21250648}.
		
		Karthigeyan K.P., Flanigan C., Machado D.J., Kiziltas A.A., Janies D.A., Chen J., Cooke D., Lee M.V., Saif L.J., Henegar S., Jahnes J., Mielewski D.F., an dKwiek J.J. (2021) Heat efficiently inactivates coronaviruses inside vehicles. \textit{bioRxiv}. DOI: \href{https://doi.org/10.1101/2021.09.08.459486}{10.1101/2021.09.08.459486}
		
		\end{hangparas}
	}